shared/fs/common/security-prefs.js

```diff
@@ -0,0 +1,123 @@
+// @flow
```

Comment:
The rendering in this file mostly stayed the same as shared/folders/install/security-prefs.desktop.js, except that I added a simple second screen after kext is successfully installed. The state handling logic is moved into the container.